test/tests.js

```diff
@@ -117,7 +117,7 @@ QUnit.test('Call to read cookie when there is another unrelated cookie with malf
 	document.cookie = 'c=v';
 	assert.strictEqual(Cookies.get('c'), 'v', 'should not throw a URI malformed exception when retrieving a single cookie');
 	assert.deepEqual(Cookies.get(), { c: 'v' }, 'should not throw a URI malformed exception when retrieving all cookies');
-	Cookies.withConverter(unescape).remove('invalid');
+	document.cookie = 'invalid=foo; expires=Thu, 01 Jan 1970 00:00:00 GMT';
```

Comment:
Why this change?

Reply:
Oh, probably due to [this comment](https://github.com/js-cookie/js-cookie/pull/409#issuecomment-366504267). It's been so long I lost context of this PR sorry :D